pkg/accountdb/dbo/user.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package dbo
+
+import (
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+	"storj.io/storj/pkg/accountdb/dbx"
+	"time"
+)
+
+// Database object that describes User entity
+type User struct {
+	id uuid.UUID
+
+	firstName string
+	lastName string
+	email string
+	password string
+
+	createdAt time.Time
+}
+
+func (d User) Id() uuid.UUID {
```

Comment:
No ned for the methods, make the fields public instead. Since these types will be mostly internal to satellite, there shouldn't be too much problem with changing declarations.

Reply:
dbo's will be changed a bit in future, when more entities will be added. Could we wait a bit with this change?

Reply:
I'm not quite following, this would mean that instead of having to only change methods you would also need to change fields?